themes/default/content/blog/pulumi-release-notes-103/index.md

```diff
@@ -0,0 +1,105 @@
+
+### Pulumi ESC Key-Value Table Editor
+
+We've rolled out the [Pulumi ESC Key-Value Table Editor](/blog/esc-key-value-table-editor-launch/), offering a novel, user-friendly interface for managing secrets and configurations. This addition complements the existing YAML-based editor, providing intuitive and familiar experiences for your workflows with secrets and configurations. Table Editor supports capabilities including CRUD operations for secrets and plaintext values, secret decryption, [environment imports](/docs/esc/environments/#importing-other-environments), variable projections, and [interpolations](/docs/esc/environments/#interpolating-values).
+
+![Key-Value Table Editor](/blog/esc-key-value-table-editor-launch/esc-key-value-table-editor.png)
+
+### Pulumi ESC YAML Code Editor Enhancements
+
+We've upgraded the Pulumi ESC YAML editor with features aimed at simplifying the environment authoring process and enhancing developer productivity. Key enhancements include auto-complete for faster coding, enhanced diagnostics offering deeper error insights, and hover documentation for easy access to information, reducing the need for context switching. Additionally, we have added support for accessing [contextual information](/docs/esc/environments/#pulumi-contextual-information) about the current user and current environments. This support can be used to fine-tune OIDC claims and enhance audit logs. For more details, check out our [blog post](/blog/esc-editor-enhancements/). 
+
+{{< video title="Pulumi ESC editor enhancements demo" src="https://www.pulumi.com/uploads/esc-editor-enhancements-demo.mp4" autoplay="true" loop="true" >}}
+
+### Pulumi ESC OIDC customization 
+
+Pulumi ESC now offers [subject customization](/docs/pulumi-cloud/oidc/azure/#subject-customization) for OIDC tokens, enhancing security across AWS, GCP and Azure by aligning federated credentials with specific identifiers. By configuring the `subjectAttributes` setting, users can include specific environment, user, and organization information in the [OIDC](/docs/pulumi-cloud/oidc/) token subject claim, offering more granular control and customization for cloud resource access. This capability is especially valuable for customers using Azure, where subject claims are required to precisely match the string specified in the policy.
+
+### Pulumi ESC Audit Logs
+
+Pulumi ESC now features comprehensive Audit Logs for all Create, Read, Open, Update, and Delete (CRUD) operations, strengthening your cloud security framework. This critical addition to Pulumi ESC enables organizations to closely monitor and audit their cloud infrastructure changes, ensuring enhanced oversight and security compliance. Through detailed logging, teams gain valuable insights into their operational activities, facilitating improved governance and accountability across cloud resources.
+
+## Core
+
+### Dev Releases for the Pulumi CLI and SDKs
```

Comment:
While we're summarizing other blog posts, can we also add another one for Node.js Monorepos? Basically summarizing and linking to https://www.pulumi.com/blog/nx-monorepo/

Reply:
Usually, we add only 'new' releases to this blog. Did we do anything to support node monorepos, or it is more of a 'how-to' guide?

Reply:
> Did we do anything to support node monorepos

Yes, we made several product enhancements to enable this (e.g. https://github.com/pulumi/pulumi/issues/12688, https://github.com/pulumi/pulumi/issues/8865)

Reply:
Is this summary good? Anything you'd add? 

Title: Node.js Monorepo Support 
Summary: Pulumi now seamlessly integrates with Node.js monorepos using tools like Nx. We've enhanced Pulumi to support yarn/npm workspaces, widely used in Node.js monorepos. This enhancement supports npm and Yarn workspaces within monorepos, facilitating better project management by automatically handling dependencies and build processes. This integration allows Pulumi infrastructure codes to coexist with application codes in the same repository, improving development workflow efficiency by using Nx's capabilities to manage build and deploy dependencies effectively.  Check out the [blog post](/blog/nx-monorepo/) for more details.

Reply:
This looks good to me